### DISCUSSION TITLE
Notify in Skype

### DISCUSSION THREAD

**Discussion by IWIZ071** [fastlane/fastlane#18790] [Q&A]
How to send success/fail messages to skype? as we do for slack.

**Comment by Shawn451** [comment]
You should look at Azure Functions as your platform, instead. Azure Functions would allow you to make the API calls you need. Additionally, you can set it to run via a trigger (timer or http request).

Instead of skype-http, check out sky-web-plus  It's basically a port of several other packages already out there, but is getting regular love from its developer (so, hopefully stable and up-to-date) and appears to do what you need. As the developer is active, you can reach out to him/her with any specific questions.

Azure Bot Framework would not be a good fit for this purpose, alone.

Hope of help!

[myloweslife](https://www.myloweslife.win/)
